public/app/routes/routes.ts

```diff
@@ -47,7 +28,8 @@ export function setupAngularRoutes($routeProvider: route.IRouteProvider, $locati
       routeInfo: DashboardRouteInfo.Home,
       reloadOnSearch: false,
       resolve: {
-        component: () => DashboardPage,
+        component: () =>
+          import(/* webpackChunkName: "DashboardPage" */ '../features/dashboard/containers/DashboardPage'),
```

Comment:
this line is duplicated in 5 spots, could be good to extract that into its own function.

Reply:
Since we are dynamically importing every React route now, what about changing the ReactContainer implementation so that it renders `ReactRouteLoader` component (would be responsible for lazy loading the chunk and loading feedback using `Suspense`)?

So instead of having `component: () => import....` in route definition we would have: 
```
resolve: {
  componentPath: "path",
  chunkName: "ChunkName"
}
```

Reply:
Unfortunately webpack needs to know the import path at compile time, so as far as I know there's currently no way to make this setup work (though there might be an existing plugin for webpack that supports this?)